libraries/chain/controller.cpp

```diff
@@ -2592,20 +2592,51 @@ block_state_ptr controller::finalize_block( const signer_callback_type& signer_c
 
```

Comment:
`finalize_block` should take `signer_callback_type` by value.

Reply:
Done.